### DISCUSSION TITLE
Add location from its coordinates

### DISCUSSION THREAD

**Discussion by EdgarOns** [breezy-weather/breezy-weather#1339] [Ideas]
### Describe your idea

In order not to depend on a limited database, it would be ideal to modernize the location search option through its geographical coordinates. This would give us the opportunity to obtain the weather forecast for any corner of the planet and would compete better against other applications in its range.

Para no depender de una base de datos limitada, sería idóneo modernizar la opción de búsqueda de ubicaciones a través de sus coordenadas geográficas. Esto nos daría la oportunidad de obtener la previsión meteorológica para cualquier rincón del planeta y competiría mejor contra otras aplicaciones de su rango.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Duplicate of #10
